massgov/openmass
# DP-32273: Include organization parent in the dataLayer

**Description:**
See ticket


**Jira:** (Skip unless you are MA staff)
DP-32273


**To Test:**

- Go to an organization page with an organization parent, such as `/orgs/executive-office-of-elder-affairs`
- View source
- Verify that the dataLayer contains a new attribute named `org_parent_node_id` with a single integer value.


---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
